instances/bodySubstance/blood.jsonld

```diff
@@ -2,9 +2,9 @@
   "@context": {
     "@vocab": "https://openminds.ebrains.eu/vocab/"
   },
-  "@id": "https://openminds.ebrains.eu/instances/UBERONParcellation/blood",
-  "@type": "https://openminds.ebrains.eu/controlledTerms/UBERONParcellation",
-  "definition":  "'Blood' is a body fluid. It runs through the circulatory system of vertebrates and delivers necessary substances such as nutrients and oxygen to the cells, and transports metabolic waste products away from those same cells.",
+  "@id": "https://openminds.ebrains.eu/instances/bodySubstance/blood",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/BodySubstance",
+  "definition": "'Blood' is a body fluid in the circulatory system of vertebrates that transports substances to and from cells (e.g. nutrients, oxygen or metabolic waste products). [adapted from Wikipedia]",
```

Comment:
"adapted from Wikipedia" should be a hyperlink to https://en.wikipedia.org/wiki/Blood